src/drivers/config.ts

```diff
@@ -0,0 +1,58 @@
+/** Copyright © 2024-present P0 Security
+
+This file is part of @p0security/cli
+
+@p0security/cli is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 3 of the License.
+
+@p0security/cli is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with @p0security/cli. If not, see <https://www.gnu.org/licenses/>.
+**/
+import { Config } from "../types/org";
+import { P0_PATH } from "../util";
+import { bootstrapConfig } from "./env";
+import { print2 } from "./stdio";
+import fs from "fs/promises";
+import path from "path";
+
+export const CONFIG_FILE_PATH = path.join(P0_PATH, "config.json");
+
+export let tenantConfig = bootstrapConfig;
```

Comment:
A mutable variable at the module-level will lead to problems imo. We will have the wrong config in it, e.g. someone makes a change that overwrites its value directly.

Can this be accessed via a getter?

```
let tenantConfig = bootstrapConfig;
...
export const getTenantConfig = () => tenantConfig
```

Reply:
Done